core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -1223,6 +1234,7 @@ public ClientDependencyInfo getClientDependencyInfo(int dependencyId)
   public ClientFileInfo getClientFileInfo(int fid) {
     synchronized (mRootLock) {
       Inode inode = mFileIdToInodes.get(fid);
+      mMasterSource.incGetFileStatusOps();
```

Comment:
Why we do this here, since the method is not guaranteed to be successful (ie. inode could be null). Its not quite in line with the other `inc*Ops` if we are only incrementing for successes.